barretenberg_static_lib/src/composer.rs

```diff
@@ -478,75 +464,74 @@ mod test {
 
     //     test_circuit(constraint_system, vec![case_1]);
     // }
```

Comment:
Can we delete this schnorr test in the lines above as well or update the values and uncomment the code? I just noticed this comment above the test: 
```
    // TODO: an added asset statement was added, making this test fail
    // TODO: we can regenerate a signature using the new code
```
which is one of the errors I ran into when I updated the backend a couple months ago and has been fixed now. However, this test does a full prove and verify rather than simply interop with barretenberg like in the `schnorr.rs` file

Reply:
Because we updated the pedersen tests to work I'm leaning towards updating this schnorr test and uncommenting it

Reply:
Schnorr test looks to be passing now so resolving this convo